### PR TITLE
remove-feedback

### PR DESCRIPTION
**Pull request by dkim32**
# Pull Request/Issue Resolution

**Description of Change:**
Removing all references to feedback feature as we sunsetted it 7/31/19.

**Reason for Change:**
Feature is sunsetted and no longer available to customers. 

Closes #**ISSUE_NUMBER_HERE**

### Is this change associated with a Braze feature/product release?
- [ ] Yes (__Feature Release Date:__)
- [ ] No

> If yes, please note the date of the feature release.

---
---

## PR Checklist
- [ ] Ensure you have completed [our CLA](https://www.braze.com/docs/cla/).
- [ ] Tag @EmilyNecciai as a reviewer when the your work is _done and ready to be reviewed for merge_.
- [ ] Consult the [Docs Text Formatting Guide](https://github.com/Appboy/success/wiki/Docs-Text-Formatting-Guide).
- [ ] Consult the [Docs Writing Style Guide & Best Practices](https://github.com/Appboy/success/wiki/Writing-Style-Guide-&-Best-Practices).
- [ ] Tag others as Reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `config/nginx.conf.erb`.

## ATTENTION: REVIEWERS OF THIS PR
- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Preview all changes in the linked Heroku environment (click `View deployment` button below, then "Docs". A `502` error just means you should refresh until you see the Docs Home page.
---
---

### CODE REVIEW

**Comment by dkim32** [comment]
@Bucimis @radixdev lmk what we should update for the customization pages as it appears that the code block refers to the default styling for feedback.

File: _docs/_developer_guide/platform_integration_guides/ios/customer_feedback.md [hunk 1]
```diff
@@ -3,6 +3,7 @@ nav_title: Customer Feedback
 platform: iOS
 page_order: 7
 search_rank: 5
+hidden: true
 ---
 # Customer Feedback
```
Comment (EmilyNecciai) [inline _docs/_developer_guide/platform_integration_guides/ios/customer_feedback.md]: We should put warnings about the deprecation on each of these pages, in the event someone finds them or clicks a link from an old email or something. 
```
{% alert Update %}
Customer Feedback is no longer supported. [Learn more about this and other deprecated features here]({{ site.baseurl }}/help/release_notes/deprecations/#feedback).
{% endalert %}
```

**Comment by Bucimis** [comment]
@dkim32 could you point to the customization pages you're referring to?

**Comment by dkim32** [comment]
> @dkim32 could you point to the customization pages you're referring to?

https://www.braze.com/docs/developer_guide/platform_integration_guides/android/news_feed/customization/

**Pull request merged**